python/pyhail/context.py

```diff
@@ -10,7 +25,10 @@ class HailContext(object):
     :param SparkContext sc: The pyspark context.
     """
 
-    def __init__(self, sc):
+    def __init__(self, sc=None, log=None, quiet=False, append=False,
```

Comment:
Why not have `log = "hail.log"` like in the Main Options?  Then the log/quiet test below is unnecessary.